cns/service/main.go

```diff
@@ -503,6 +503,7 @@ func main() {
 	cnsconfig, err := configuration.ReadConfig(cmdLineConfigPath)
 	if err != nil {
 		logger.Errorf("[Azure CNS] Error reading cns config: %v", err)
+		cnsconfig = &configuration.CNSConfig{}
```

Comment:
We should make it apparent in the error log line that a default will be used. I'm also curious if we should only use a default in cases where the file is missing. If, for example, the file is malformed, maybe it's better to still crash such that orchestrators take action.

Reply:
agree and updated